src/lib.rs

```diff
@@ -941,7 +964,204 @@ impl UInt {
+    ///
+    /// # Examples
+    ///
+    /// Basic usage:
+    ///
+    /// ```
+    /// # use js_int::UInt;
+    /// assert_eq!(UInt::from(100u32).wrapping_div(UInt::from(10u32)), UInt::from(10u32));
+    /// ```
+    #[must_use]
+    pub fn wrapping_div(self, rhs: Self) -> Self {
+        Self(self.0.wrapping_div(rhs.0))
+    }
+
+    /// Wrapping (modular) remainder. Computes `self % rhs`. Wrapped remainder calculation on
+    /// unsigned types is just the regular remainder calculation. There's no way wrapping could ever
+    /// happen. This function exists, so that all operations are accounted for in the wrapping
+    /// operations.
+    ///
+    /// # Examples
+    ///
+    /// Basic usage:
+    ///
+    /// ```
+    /// # use js_int::UInt;
+    /// assert_eq!(UInt::from(100u32).wrapping_rem(UInt::from(10u32)), UInt::from(0u32));
+    /// ```
+    #[must_use]
+    pub fn wrapping_rem(self, rhs: Self) -> Self {
+        Self::new_wrapping(self.0.wrapping_rem(rhs.0))
+    }
+
+    /// Wrapping (modular) exponentiation. Computes `self.pow(rhs)`, Wrapping around the boundary
+    /// of the type.
+    ///
+    /// # Examples
+    ///
+    /// Basic usage:
+    ///
+    /// ```
+    /// # use js_int::UInt;
+    /// assert_eq!(UInt::from(100u32).wrapping_rem(UInt::from(10u32)), UInt::from(0u32));
+    /// ```
+    #[must_use]
+    pub fn wrapping_pow(self, exp: u32) -> Self {
+        Self::new_wrapping(self.0.wrapping_pow(exp))
+    }
+
+    /// Calculates `self` + `rhs`
+    ///
+    /// Returns a tuple of the addition along with a boolean indicating whether an arithmetic
+    /// overflow would occur. If an overflow would have occurred then the wrapped value is returned.
+    ///
+    /// # Examples
+    ///
+    /// Basic usage:
+    ///
+    /// ```
+    /// # use js_int::UInt;
+    /// assert_eq!(UInt::from(100u32).overflowing_add(UInt::from(1u32)), (UInt::from(101u32), false));
+    /// assert_eq!(UInt::MAX.overflowing_add(UInt::from(1u32)), (UInt::from(0u32), true));
+    /// ```
+    #[must_use]
+    pub fn overflowing_add(self, rhs: Self) -> (Self, bool) {
+        Self::new_overflowing(self.0.wrapping_add(rhs.0))
+    }
+
+    /// Calculates `self` - `rhs`
+    ///
+    /// Returns a tuple of the subtraction along with a boolean indicating whether an arithmetic
+    /// overflow would occur. If an overflow would have occurred then the wrapped value is returned.
+    ///
+    /// # Examples
+    ///
+    /// Basic usage:
+    ///
+    /// ```
+    /// # use js_int::UInt;
+    /// assert_eq!(UInt::from(5u32).overflowing_sub(UInt::from(2u32)), (UInt::from(3u32), false));
+    /// assert_eq!(UInt::from(0u32).overflowing_sub(UInt::from(1u32)), (UInt::MAX, true));
+    /// ```
+    #[must_use]
+    pub fn overflowing_sub(self, rhs: Self) -> (Self, bool) {
+        Self::new_overflowing(self.0.wrapping_sub(rhs.0))
+    }
+
+    /// Calculates multiplication of `self` and `rhs`
```

Comment:
Why is this worded differently from the others?

Reply:
I followed std::u64 that's how the docs for the stdlib are. Should I change it to `Overflowing multiplication. Computes 'self * rhs' wrapping at the type boundary`?

Reply:
Yeah, but with a comma before "wrapping" ;)